react/react.d.ts

```diff
@@ -3,7 +3,7 @@
 // Definitions by: Asana <https://asana.com>, AssureSign <http://www.assuresign.com>
 // Definitions: https://github.com/borisyankov/DefinitelyTyped
 
-declare module "react" {
+declare module React {
```

Comment:
1) Declaring a global React module means that React may be referenced even when it hasn't been explicitly imported.
2) IntelliJ doesn't like it (I don't personally use IntelliJ with react/typescript, but see https://github.com/borisyankov/DefinitelyTyped/pull/3959 for more context)